AppCheckCore.podspec

```diff
@@ -1,5 +1,5 @@
 Pod::Spec.new do |s|
-  s.name             = 'AppCheck'
+  s.name             = 'AppCheckCore'
```

Comment:
Why `AppCheckCore` vs `AppCheck`?

Reply:
I was trying to avoid naming conflicts and confusion with `FirebaseAppCheck`, which is sometimes just referred to as `AppCheck` ([example](https://github.com/firebase/firebase-ios-sdk/blob/2937104f9b6adbc98ef1603b258c8251f2e09304/.github/workflows/app_check.yml#L68)). Another alternative would be to rename existing non-prefixed `AppCheck` references to `FirebaseAppCheck`. Any preferences or recommendations?

Reply:
This is ultimately a product management decision. My not very strong preference is that this be AppCheck and we're consistent about `FirebaseAppCheck` for the wrapper.

I'm ok with this for now, with the potential to string change later.

Reply:
Our use of `FirebaseAppCheck` should now be consistent (as of PR #11359, thanks for the review!) and I've reverted to `AppCheck` in this PR. This should 🤞 make it easy to rename if we go with a different name than `AppCheck` and avoid confusion in the meantime.